code/modules/clothing/suits/wintercoats.dm

```diff
@@ -17,7 +17,7 @@
 		/obj/item/tank/internals/plasmaman,
 		/obj/item/toy,
 )
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 10, RAD = 0, FIRE = 0, ACID = 0)
+	armor = list(MELEE = 5, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 10, RAD = 0, FIRE = 0, ACID = 0)
```

Comment:
this random balance change isnt even on the winter hood counterpart and some other armor changes arent reflect on hoods as well, they should all be consistent between coat and hood